content/docs/2.7/operate/cluster.md

```diff
@@ -65,6 +65,19 @@ $env:KEDA_HTTP_DEFAULT_TIMEOUT=1000
 
 All applicable scalers will use this timeout. Setting a per-scaler timeout is currently unsupported.
 
+## HTTP connection disable keep alive
+You can override this default by setting the `DISABLE-KEEP-ALIVES` environment variable to true. For example, on Linux/Mac/Windows WSL2 operating systems, you'd use this command to set to true:
+```shell
+$env:DISABLE-KEEP-ALIVES=true
+```
+
+And on Windows Powershell, you'd use this command:
+
+```shell
+$env:DISABLE-KEEP-ALIVES=true
```

Comment:
```suggestion
$env:KEDA_HTTP_DISABLE_KEEP_ALIVE=true
```

Reply:
@JorTurFer, I have change it, please review again, thanks